### DISCUSSION TITLE
Last element or object

### DISCUSSION THREAD

**Discussion by Kuryuuki** [getgauge/taiko#2400] [General]
Hi, I would like to know how do I select the last element/object. I just found a constant from first to, last no

Examples:
await tap('@', {exactMatch: true}, LAST OBJECT) 
await click('@', {exactMatch: true}, LAST OBJECT)

**Comment by Kuryuuki** [comment]
"selector
A selector to search for element to click / coordinates of the elemets to click on. If there are multiple elements satisfying the selector, the first will be clicked."

couldn't there be a function inside taiko to always be able to call the last option?

**Comment by Kuryuuki** [comment]
Somebody help me,  pls

**Comment by NivedhaSenthil** [comment]
There is no direct method to perform action on the last matching element but you can iterate over the list to of matching elements to perform action on desired element.  Please refer https://docs.taiko.dev/api/elementwrapper/#elements.

I would also suggest trying a more specific selector to correctly pick the element to be actioned on rather than iterating over to last element as this can avoid any flakiness related to element ordering.

**Comment by Kuryuuki** [comment]
@NivedhaSenthil could you give me an example of how to click on the last "test" element?

await click("teste") final element

**Comment by Kuryuuki** [comment]
Example jquery $('span.mention').textContent="@"

How I set to click($('span.mention').textContent="@") last element?

**Reply by zabil** [reply]
Please try using css selector [`last-of-type`](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-of-type) for e.g.

```
click($('span.mention :last-of-type'))
```

**Reply by Kuryuuki** [reply]
the button click return always the first element, even using the "last-of-type"
